mula/scheduler/models/schedule.py

```diff
@@ -47,6 +74,16 @@ def validate_schedule(cls, value: str) -> str:
         except Exception as exc:
             raise ValueError(f"Invalid cron expression: {value}") from exc
 
+    @classmethod
+    def model_validate(cls, data):
```

Comment:
See earlier remark about model_post_init, which could make this block unnecessary